cloudify_cli/replace_certificates_config.py

```diff
@@ -162,18 +163,19 @@ def replace_certificates(self):
         self._handle_new_ca_certs()
         # Passing only the new CA cert to the agents
         self._pass_new_ca_certs_to_agents(bundle=False)
-        self.logger.info('Validating status is healthy')
+        self.logger.info('\nValidating status is healthy')
         self._validate_status_ok()
-        self.logger.info('\nSuccessfully replaced certificates')
+        self.logger.info('Successfully replaced certificates')
 
     def _handle_new_ca_certs(self):
         """Replace the CLI and client CA certs"""
         if env.profile.rest_certificate:
             new_manager_ca = self.config_dict['manager'].get('new_ca_cert')
             if new_manager_ca:
-                env.profile.rest_certificate = new_manager_ca
+                expanded_path = expanduser(new_manager_ca)
+                env.profile.rest_certificate = expanduser(expanded_path)
```

Comment:
no need to expand it twice :P

Reply:
Thanks! Forgot to delete it after creating the variable